hass_nabucasa/iot.py

```diff
@@ -137,6 +137,16 @@ async def _async_handle_handler_message(self, message: dict[str, Any]) -> None:
         with suppress(ConnectionResetError):
             await self.client.send_json(response)
 
+    async def _connected(self) -> None:
+        """Handle connected."""
+        await super()._connected()
```

Comment:
Should we have a guard here and in `CloudIoT.__connected` that checks if the state is `STATE_CONNECTED` and if it is, it returns early?

Reply:
The caller already checks the state before all calls to `_connected` and `_disconnected`. We should not add additional state checks in the methods, but we could consider moving the state checks to the methods in the future.